src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -451,6 +452,15 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly filteredAlterationsAsUniquePatientKeyArrays = remoteData({
+        await: () => [
+            this.filteredAlterations
+        ],
+        invoke: async() => {
+            return _.mapValues(this.filteredAlterations.result, (mutations: Mutation[]) => _.map(mutations, 'uniquePatientKey'));
+        }
```

Comment:
minor but for map i like to use (whatever)=>whatever.property
to get typechecking on the prop name